src/main/java/echoserver/EchoServer.java

```diff
@@ -1,5 +1,8 @@
 package echoserver;
 
+import server.*;
```

Comment:
I like to avoid wildcard imports on most occasions. I even wrote a blog post on it.

https://blog.damonkelley.me/2016/06/29/import-star/

There are some reasonable exceptions though, like importing all of the JUnit assertions or hamcrest matcher functions code.